class.photon.php

```diff
@@ -290,6 +290,23 @@ public static function filter_the_content( $content ) {
 
 						// Supplant the original source value with our Photon URL
 						$photon_url = esc_url( $photon_url );
+						$zoom_url_2 = add_query_arg( 'zoom', 2, $photon_url );
```

Comment:
Rather than creating three new variables, it's probably easier to just do these `$zoom_url_#` variables inline in the array declaration below.